openshift/dynamic/client.py

```diff
@@ -335,6 +335,8 @@ def validate(self, definition, version=None, strict=False):
             kubernetes_validate.validate(definition, version, strict)
         except kubernetes_validate.utils.ValidationError as e:
             errors.append("resource definition validation error at %s: %s" % ('.'.join([str(item) for item in e.path]), e.message))  # noqa: B306
+        except kubernetes_validate.utils.VersionNotSupportedError as e:
```

Comment:
Do we need to update recommended version bounds to make sure we have this exception class?

Reply:
I haven't tested that - I'm not sure how lazy python is at evaluating this kind of thing.

Worth testing though.

Reply:
I'm good to merge if you think we're good with this issue.

Reply:
I still haven't tested that. I'll do that now.

Reply:
Rudimentary testing suggests this would indeed fail :(

Reply:
I have a fix that I'll be able to test with the ansible test suite once openshift reports the right version :)

Reply:
I think it should be ok now